src/Groups/perm.jl

```diff
@@ -162,6 +162,30 @@ julia> length(moved_points(gen(s, 1)))
 """
 @gapattribute moved_points(x::Union{PermGroupElem,PermGroup}) = Vector{Int}(GAP.Globals.MovedPoints(GapObj(x)))
 
+@doc raw"""
+    fixed_points(G::PermGroup) -> Vector{Int}
+
+Return the list of fixed points of the permutation group `G`.  
+A fixed point is an integer in `1:degree(G)` that is not moved by any element of `G`.
+
+# Examples
+```jldoctest
+julia> g = symmetric_group(4);  s = sylow_subgroup(g, 3)[1];
+
+julia> fixed_points(s)
+Int64[]
+
+julia> fixed_points(one(s))
+4-element Vector{Int64}:
+ 1
+ 2
+ 3
+ 4
+```
+"""
+fixed_points(G::PermGroup) = setdiff!(collect(1:degree(G)), moved_points(G))
```

Comment:
```suggestion
fixed_points(G::PermGroup) = setdiff(1:degree(G), moved_points(G))
```
I did a small benchmark and it seems as if the original version of this was indeed better than my suggetsion, so this should be changed back.
```julia
julia> f1(n, x) = setdiff(1:n, x)
f1 (generic function with 1 method)

julia> f2(n, x) = setdiff(collect(1:n), x)
f2 (generic function with 1 method)

julia> f3(n, x) = setdiff!(1:n, x)
f3 (generic function with 1 method)

julia> f4(n, x) = setdiff!(collect(1:n), x)
f4 (generic function with 1 method)

julia> @b f1($(10), $([1,3,5,6]))
382.658 ns (6 allocs: 448 bytes)

julia> @b f2($(10), $([1,3,5,6]))
416.382 ns (8 allocs: 592 bytes)

julia> @b f3($(10), $([1,3,5,6]))
ERROR: CanonicalIndexError: setindex! not defined for UnitRange{Int64}
[...]

julia> @b f4($(10), $([1,3,5,6]))
398.235 ns (7 allocs: 544 bytes)
```